kafka/src/test/groovy/io/micronaut/configuration/kafka/annotation/KafkaListenerSpec.groovy

```diff
@@ -293,7 +293,7 @@ class KafkaListenerSpec extends AbstractEmbeddedServerSpec {
     }
 
     @Requires(property = 'spec.name', value = 'KafkaListenerSpec')
-    @KafkaListener(clientId = "kafka-consumer-with-configurable-number-of-threads", threads = '${my.thread.count}')
+    @KafkaListener(clientId = "kafka-consumer-with-configurable-number-of-threads", threadsValue = '${my.thread.count}')
```

Comment:
I'd like to see some additional test cases to also exercise setting the `threads` attribute, and then setting both `threadsValue` and `threads` at the same time.